{"name": "test_upload_image_byid", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'slide-1.jpg'", "trace": "MPet = <models.pet.Pet object at 0x000001EB239DD070>\n\n    def test_upload_image_byid(MPet):\n        mylogger.info(\"test for upload image by id\")\n>       files = {'upload_file': open('slide-1.jpg', 'rb')}\nE       FileNotFoundError: [Errno 2] No such file or directory: 'slide-1.jpg'\n\npet_test.py:131: FileNotFoundError"}, "start": 1659702178861, "stop": 1659702178862, "uuid": "436d70b4-bf1d-4732-99f0-6c6620d97011", "historyId": "241253d1ad759c89523cd36692108b7b", "testCaseId": "cadf913662485419543a43ab707e4a6b", "fullName": "pet_test#test_upload_image_byid", "labels": [{"name": "suite", "value": "pet_test"}, {"name": "host", "value": "DESKTOP-6448GON"}, {"name": "thread", "value": "17288-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pet_test"}]}